src/clj/game/cards/assets.clj

```diff
@@ -979,6 +979,17 @@
                                                    (trigger-event state side :no-trash card))))}
                                card targets))}}
 
+   "NGO Front"
+   {:advanceable :always
+    :abilities [{:advance-counter-cost 1
+                 :effect (effect (trash card) (gain :credit 5))
```

Comment:
Add a final parameter `{:cause :ability-cost}` to the `trash` calls to indicate that we're paying an ability cost as the reason for the trash.